docs/dsp-tools-xmlupload.md

```diff
@@ -100,102 +112,64 @@ permission `special-permission` can only be viewed by `ProjectAdmin`s:
-    xsi:schemaLocation="https://dasch.swiss/schema https://raw.githubusercontent.com/dasch-swiss/dsp-tools/main/knora/dsplib/schemas/data.xsd"
-    shortcode="0806"
-    default-ontology="webern">
-  
-  <permissions id="res-default">
-    <allow group="UnknownUser">RV</allow>
+```
+<permissions id="res-default">
+    <allow group="UnknownUser">V</allow>
     <allow group="KnownUser">V</allow>
-    <allow group="Creator">CR</allow>
+    <allow group="ProjectMember">D</allow>
     <allow group="ProjectAdmin">CR</allow>
-  </permissions>
-  
-  <permissions id="res-restricted">
-    <allow group="KnownUser">V</allow>
     <allow group="Creator">CR</allow>
+</permissions>
+<permissions id="res-restricted">
+    <allow group="ProjectMember">M</allow>
     <allow group="ProjectAdmin">CR</allow>
-  </permissions>
-  
-  <permissions id="prop-default">
+    <allow group="Creator">CR</allow>
+</permissions>
+<permissions id="prop-default">
     <allow group="UnknownUser">V</allow>
     <allow group="KnownUser">V</allow>
-    <allow group="Creator">CR</allow>
+    <allow group="ProjectMember">D</allow>
     <allow group="ProjectAdmin">CR</allow>
-  </permissions>
-  
-  <permissions id="prop-restricted">
-    <allow group="KnownUser">RV</allow>
     <allow group="Creator">CR</allow>
+</permissions>
+<permissions id="prop-restricted">
+    <allow group="ProjectMember">M</allow>
     <allow group="ProjectAdmin">CR</allow>
-  </permissions>
-  ...
-</knora>
+    <allow group="Creator">CR</allow>
+</permissions>
 ```
 
 
-### How to use the `permissions` attribute in resources/properties
+### The `permissions` attribute in resources/properties
 
-Based on the permissions section of the above example, there are different ways how to grant permissions to a resource
-and its properties. It is important to note that a resource doesn't inherit its permissions to its properties. Each 
-property must have its own permissions. So, in the following example, the bitstreams don't inherit the permissions from their 
-resource:
+The permission IDs of the above example can now be used as `permissions` attribute in the `<resource>` and 
+`<xyz-prop>` tags. It is important to note that a resource doesn't inherit its permissions to its properties. 
+Each property must have its own permissions. So, in the following example, the bitstreams don't inherit the permissions 
+from their resource:
 
 ```
 <resource ...>
-    <bitstream permissions="prop-default">
-        postcards/images/EURUS015a.jpg
-    </bitstream>
+    <bitstream permissions="prop-default">images/EURUS015a.jpg</bitstream>
 </resource>
 <resource ...>
-    <bitstream permissions="prop-restricted">
-        postcards/images/EURUS015a.jpg
-    </bitstream>
+    <bitstream permissions="prop-restricted">images/EURUS015a.jpg</bitstream>
 </resource>
 <resource ...>
-    <bitstream>
-        postcards/images/EURUS015a.jpg
-    </bitstream>
+    <bitstream>images/EURUS015a.jpg</bitstream>
 </resource>
 ```
```

Comment:
I'm not a fan of examples showing what does _not_ work. Instead, examples should always show how things work - and the best way of doing something.